tests/remotes/azure.py

```diff
@@ -30,7 +33,7 @@ def get_url():
 def azure():
     if not Azure.should_test():
         pytest.skip("no azure running")
-    yield Azure()
+    yield Azure(Azure.get_url())
```

Comment:
This `CLS(CLS.get_url())` mess will be removed once no tests use it. Likely in this PR.